iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java

```diff
@@ -336,7 +336,7 @@ public static void voltageLevelRemovingEquipmentsLeftReport(Reporter reporter, S
     public static void notFoundIdentifiableReport(Reporter reporter, String identifiableId) {
         reporter.report(Report.builder()
```

Comment:
This method is too generic to have a message linked to couplers.

Reply:
I have created a new method to handle that